landingpage/src/components/AlterConfigForName.tsx

```diff
@@ -1,30 +1,36 @@
-import { util } from 'plebnames';
-import React, { useState } from 'react';
+import { PlebNameData, util } from 'plebnames';
+import React, { useEffect, useState } from 'react';
 import { FaCopy } from 'react-icons/fa6';
 import DropDownContent from './HeaderWithSearch/DropDownContent';
 
 /** Allowed Keys */
-type AlterOptionKey =
-	| 'nostr'
-	| 'website'
-	| 'owner'
-	| 'lightningAddress'
+export type AlterOptionKey = keyof PlebNameData
 	| 'any';
 
+
+type Output = { key: AlterOptionKey, value: string };
+
+
 const AlterConfigForName: React.FC<{
 	queryString: string;
 	currentOwner: string;
+	configKey?: AlterOptionKey;
```

Comment:
that's a good option for having a default (e.g. nostr by default and website when coming from a browser extension and website is not set)